vendor/github.com/opencontainers/selinux/go-selinux/label/label_selinux.go

```diff
@@ -90,14 +104,36 @@ func SetProcessLabel(processLabel string) error {
 	return selinux.SetExecLabel(processLabel)
 }
 
+// SetSocketLabel takes a process label and tells the kernel to assign the
+// label to the next socket that gets created
+func SetSocketLabel(processLabel string) error {
```

Comment:
I'm not an SELinux expert (so definitely could use some eyes on this PR 😅 

I noticed this was added; not sure if this is needed / has to be used for anything in this repository?

Reply:
I'm not familiar with SELinux, either.

We don't use this function today.
Based on https://github.com/opencontainers/selinux/pull/38, this is a completely new function, so I don't think it will cause any regression.